engine/dlib/src/dlib/log.cpp

```diff
@@ -303,11 +306,16 @@ static void DoLogPlatform(LogSeverity severity, const char* output, int output_l
 #endif
 
 #ifdef __EMSCRIPTEN__
+
         //Emscripten maps stderr to console.error and stdout to console.log.
         if (severity == LOG_SEVERITY_ERROR || severity == LOG_SEVERITY_FATAL){
-            fwrite(output, 1, output_len, stderr);
```

Comment:
With this approach, each new line symbol `\n` creates a new error it makes it hard to collect such errors also it looks not that well:
![CleanShot 2024-07-02 at 18 54 01@2x](https://github.com/defold/defold/assets/2209596/c218c624-f44a-4568-b675-dc704a0af35c)